### DISCUSSION TITLE
Outlaw Rogue Blade Flurry/Blade Rush synergy

### DISCUSSION THREAD

**Discussion by ccamisa** [Hekili/hekili#851] [General]
I noticed that for Outlaw Rogue cleave priority, the suggestion is being made to cast Blade Rush *then* activate Blade Flurry but this seems like it would forego the bonus damage done by Blade Flurry on top of the Blade Rush cast.

Is there something I'm missing?

**Comment by Hekili** [comment]
Submit an issue ticket with a snapshot from when that happens.
